data/windows.yaml

```diff
@@ -3447,3 +3483,37 @@ sources:
 labels: [Execution]
 supported_os: [Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Windows#Crash_and_minidumps']
+---
+name: WindowsMsOfficeAutosave
+doc: Recovery files automatically created by Microsoft Office applications.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.appdata%%\Microsoft\Word\**'
+    - '%%users.appdata%%\Microsoft\Excel\**'
+    - '%%users.appdata%%\Microsoft\Powerpoint\**'
+    - '%%users.appdata%%\Microsoft\Publisher\**'
+    separator: '\'
+labels: [Users]
+supported_os: [Windows]
+urls: ['https://forensicswiki.xyz/wiki/index.php?title=Windows#Microsoft_Office_Autosave']
+---
+name: WindowsUserAccessLogging
+doc: |
+  User Access Logging (UAL) databases.
+
+  UAL is a local data aggregation feature (enabled by default) on Windows
+  Servers 2012 and above, recording client usage by role and product on each
+  system providing the resource. It's typically between 2 and 4 extensible
+  storage engine (ESE) databases ("Current.mdb", "SystemIdentity.mdb, and
+  "<GUID>.mdb").
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\System32\LogFiles\SUM\*.mdb'
```

Comment:
since this is a single path definition use the form `paths: ['...']`

Reply:
My oversight, fixed! 👍